### DISCUSSION TITLE
Git changes not deploying

### DISCUSSION THREAD

**Discussion by C6e6y6e** [vercel/community#3825] [Help] [answered]
### Page to Investigate

-

### Steps to Reproduce

Anyone else having an issue with changes not deploying?

**Comment by danielvaughn** [comment]
This is also happening to me. I have two projects - one that was initially deployed on project creation 2 days ago. Another project I have auto-deployed several times over the past month.

The latter can auto-deploy, even though it was significantly delayed (like 15-20 minutes I think). The former will not auto-deploy at all. It's been like an hour and I've committed multiple times. Very frustrating.

I pinged vercel-support on twitter but they haven't responded.

**Comment by C6e6y6e** [comment]
Looks like they are aware and theyre working on it. You can see their notices here https://www.vercel-status.com/

**Comment by offchan42** [comment]
Yeah I'm having the same issue. Pushes to main are not being built. I thought it was because I renamed the repo so I deleted the project and created a new one. But creating a new project doesn't resolve the issue so it must definitely be Vercel's (or someone else's) fault, not mine.

**Reply by C6e6y6e** [reply]
Looks like they are aware and theyre working on it. You can see their notices here https://www.vercel-status.com/ My deployment are going through at like 20 minute intervals. I think they are queued.

**Comment by swarnava** [comment, marked as answer]
FWIW, this issue is coming from GitHub https://www.githubstatus.com/incidents/wqv4yvlgtq62 & we are tracking here: https://www.vercel-status.com/incidents/r7md342b3d39
